Client/WebFilters/ShieldStats/Adblock/AdBlockStats.swift

```diff
@@ -136,15 +140,18 @@ class AdBlockStats: LocalAdblockResourceProtocol {
             return completion
         }
         
-        if engine.set(data: data) {
-            log.debug("Adblock file with id: \(id) deserialized successfully")
-            // Clearing the cache or checked urls.
-            // The new list can bring blocked resource that were previously set as not-blocked.
-            fifoCacheOfUrlsChecked = FifoDict()
-            completion.fill(())
-        } else {
-            log.error("Failed to deserialize adblock list with id: \(id)")
+        AdBlockStats.adblockSerialQueue.async {
```

Comment:
It would be ideal to move this serial queue directly to `AdblockRust` class but I didn't want to make a bigger rewrite for now